interfaces/cython/cantera/ck2cti.py

```diff
@@ -1933,14 +1933,15 @@ def parseTransportData(self, lines, filename, line_offset):
 
             if '!' in line:
                 line, comment = line.split('!', 1)
```

Comment:
I realize that we ultimately don't include the `comment` in the CTI file, but I think we should still pass it to the `TransportData` constructor (as a keyword argument). Will need to define `comment` in the other branch of the `if` in this case.